src/screens/CreateAccount.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react'
+import { ScrollView, StyleSheet, TouchableHighlight,
+  Text, Dimensions, View,
+} from 'react-native'
+import PropTypes from 'prop-types'
+
+import Translate from '../components/Translate'
+import styles from '../styles/common'
+
+export default class CreateAccount extends Component {
+  static propTypes = {
+    navigation: PropTypes.object.isRequired,
+  }
+
+  onPressFB = () => {
+    this.props.navigation.navigate('Facebook')
+  }
+
+  onPressSignIn = () => {
+    this.props.navigation.navigate('SignIn')
+  }
+
+  render () {
+    return (
+      <ScrollView style={[localStyles.container]} >
```

Comment:
keep in mind that we should be using `Container` on every screen, no need to do it in this PR though.